src/App.tsx

```diff
@@ -1,61 +1,139 @@
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find((user) => user.id === userId);
+
+  return foundUser || null;
+}
+
+const initialTodos = todosFromServer.map((todo) => ({
```

Comment:
Consider using a more descriptive name for `initialTodos`. For example, `todosWithUserDetails`. 🤔